cli/azd/pkg/project/framework_service_npm.go

```diff
@@ -106,3 +109,16 @@ func (np *npmProject) Build(
 		},
 	)
 }
+
+const cNodeModulesName = "node_modules"
```

Comment:
Nit: Not sure we we are prefixing the constant with `c`.  If it's a constant it will be defined as such.

Reply:
This is a convention that I somewhat established (I think I picked it up from reading the GH CLI sources).  The use of the lowercase first letter ensures that this value is not exported from the package. while still making it somewhat clear that this is a constant instead of just something like `nodeModulesName` where it's a little less clear.

Reply:
I think it is a good convention